src/HH4b/postprocessing/CreateDatacard.py

```diff
@@ -332,6 +300,20 @@
         pass_only=True,
         convert_shape_to_lnN=True,
     ),
+    # "FSRPartonShower": Syst(name="ps_fsr", prior="shape", samples=sig_keys),
+    # "ISRPartonShower": Syst(name="ps_isr", prior="shape", samples=sig_keys),
+    "scale": Syst(
+        name=f"{CMS_PARAMS_LABEL}_QCDScaleacc",
+        prior="shape",
+        samples=sig_keys,  # + ["ttbar"],  # FIXME: add back ttbar later
+        samples_corr=False,
```

Comment:
@rkansal47 should QCDScaleacc and PDFacc be correlated across the different qqHH and (separately) across the different ggHH signals rather than completely uncorrelated for each HH signal parameter point?